assets/app/view/game/companies.rb

```diff
@@ -25,6 +31,34 @@ def render_companies(owner, companies)
           *companies.map { |c| h(Company, company: c) },
         ])
       end
+
+      def render_companies_table(owner)
+        companies = owner.companies.map do |c|
+          h(Company, company: c, table: true)
+        end
+
+        table_props = {
+          style: {
+            padding: '0 0.5rem',
+            justifySelf: 'stretch',
+          },
+        }
+        row_props = {
+          style: {
+            grid: owner.player? ? '1fr / 4fr 1fr 1fr' : '1fr / 5fr 1fr',
+            justifySelf: 'stretch',
+          },
+        }
+
+        h('div#company_table', table_props, [
+          h('div.bold', row_props, [
+            h(:div, 'Company'),
+            owner.player? ? h('div.right', 'Value') : '',
+            h('div.right', 'Income'),
+          ]),
+          h(:div, [*companies]),
```

Comment:
h(:div, companies)

Reply:
> Is there a border here?
Looks like 4px black, but it should be in user-selected font-color (corporation 23) like so …
![image](https://user-images.githubusercontent.com/33390595/85883276-035a5e00-b7e1-11ea-8ee2-22ad5b20c5e3.png)
But there’s no need for explicit color anyway, inheritance should be sufficient.

Reply:
do you need to fix something?

Reply:
color just removed, please try